CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This document outlines major changes between releases.
 
+## 0.103.1 "Verification" (08 Nov 2023)
+
+An urgent 3.6.0-compatible version that contains a hotfix for the bug that
+prevents node from starting from the existing database every new dBFT epoch.
+Also, the release includes a bugfix that fails any non-zero NEO and GAS
+roundtrips from accounts with zero balance.
+
+Although the DB format wasn't changed, this release requires a full
```

Comment:
I'd say it's optional. Recommended, but can be skipped if you're ok with wrong application logs for some transactions (CN doesn't care for example). The state itself is the same.